source/includes/fundamentals/code-snippets/srv.java

```diff
@@ -11,11 +11,10 @@
 import com.mongodb.client.MongoClients;
 import com.mongodb.client.MongoDatabase;
 
-
 public class RunCommand {
     public static void main(String[] args) {
         // Replace the uri string with your MongoDB deployment's connection string
-        String uri = "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+        String uri = "mongodb://user:pass@sample.host:27017/?maxPoolSize=20&w=majority";
```

Comment:
I am convinced and agree with your thinking on everything after the @ sign, however I think user and pass should be enclosed in \<\>. I think our docs are fairly consistent that text you are expected to replace is enclosed in \<\>. With that in mind, here is my proposed change:

```suggestion
        String uri = "mongodb://<user>:<pass>@sample.host:27017/?maxPoolSize=20&w=majority";
```

I think otherwise this communicates that "user" and "pass" are your username and password, whereas [the above section says](https://github.com/mongodb/docs-java/blob/20cf0dd2a7248746e18a7f6bddff449d273b1d2a/source/fundamentals/connection.txt#L75-L76) to replace user and pass with your username and password.